test/sharness/t0235-cli-request.sh

```diff
@@ -10,23 +10,39 @@ test_description="test http requests made by cli"
 
 test_init_ipfs
 
-test_expect_success "can make http request against nc server" '
-  nc -ld 5005 > nc_out &
+test_expect_success "start nc" '
+  rm -f nc_out nc_in && mkfifo nc_in nc_out
+  # 1. Abuse cat to buffer output.
+  # 2. Put cat in a subshell so we capture the PID of nc.
+  nc -k -l 5005 < nc_in > >(cat > nc_out) &
```

Comment:
Sholudn't this be:
```suggestion
  nc -k -l 5005 < <(cat nc_in) > nc_out &
```

Reply:
No, I needed to buffer *output* because netcat wasn't (or something like that...). Really, I'm not sure what was happening. I just know that everything locked up without this.